tensorflow_serving/example/inception_export.py

```diff
@@ -100,7 +100,7 @@ def export():
     # the human readable class description for the i'th index.
     class_tensor = tf.constant([texts[s] for s in synsets])
 
-    classes = tf.contrib.lookup.index_to_string(tf.to_int64(indices),
+    classes = tf.contrib.lookup.index_to_string(tf.sub(tf.to_int64(indices), tf.constant(1, dtype=tf.int64)),
```

Comment:
nit: please format it so each line is <80 chars.
(and please squash commits when you're done)